perma_web/perma/templates/terms_of_service.html

```diff
@@ -38,15 +38,15 @@ <h1>Perma Terms Of Service</h1>
 
     <div class="col-sm-8 docs reading-body">
 
-      <h2 class="body-ah">Effective Date: September, 23, 2013</h2>
+      <h2 class="body-ah">Effective Date: January, 22 2019</h2>
```

Comment:
Wrong comma removed 😉

Reply:
And that's known as a "real dumber typo"

Reply:
A certain song about Mondays comes to mind